tests/test_ble_interface_advanced.py

```
@@ -726,30 +829,47 @@ def _stress_test_disconnects():
+                    time.sleep(0.01)
+                    continue
+                iface3._on_ble_disconnect(cast(Any, current_client.bleak_client))
                 time.sleep(0.01)
-            except Exception as e:  # noqa: BLE001 - expected during failure stress
-                logging.debug(
-                    "Expected failure during stress reconnect: %s: %s",
-                    type(e).__name__,
-                    e,
-                )
+            except (
+                Exception
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)